scripts/build_windows.cmd

```diff
@@ -21,6 +21,7 @@ where iscc
 if %errorlevel% neq 0 (echo Error: Couldn't find Inno Setup compiler. && goto :error)
 
 echo ====== Installing requirements... ======
+echo PKG = "exe" > dvc\utils\build.py
```

Comment:
Hm, not quite sure this is going to work. Have you tried running it in cmd? If not, I'll give it a shot in a few minutes.

Reply:
Yes! I've sent it to one of my friends and he confirmed that I don't need any escapes on that. But feel free to double check it to be 100% sure

Reply:
@n3hrox I don't think your friend is correct, as running that without single quotes results in
```
PKG
=
exe
```

which is wrong

Reply:
@efiop weird, I will change it then

Reply:
I was wrong. Using single quotes made it print with those quotes to the file :slightly_frowning_face: And using no quotes does work correctly, looks like I did something wrong when testing it. Sorry for that. I'll adjust it myself. Big thanks to your friend! :slightly_smiling_face: